nautobot/docs/assets/extra.css

```diff
@@ -137,3 +137,12 @@ img.copyright-logo {
     -webkit-mask-image: var(--md-admonition-icon--version-removed);
     mask-image: var(--md-admonition-icon--version-removed);
 }
+
+.md-typeset .tabbed-set {
+    border: 0.5px solid gray;
+}
+
+/* .md-typeset .tabbed-content {
+    padding-left: 1rem;
+    padding-right: 1rem;
+} */
```

Comment:
```suggestion
```

Reply:
I just got too excited about the look and feel!!!

Reply:
And then jumping to meetings.